RoddyCore/src/de/dkfz/roddy/core/ProjectLoader.groovy

```diff
@@ -494,13 +494,36 @@ class ProjectLoader {
         List<String> errors = []
 
         // Earliest check for valid input and output directories. If they are not accessible or writeable.
-        // Start with the input directory
-        errors += checkDirForReadabilityAndExecutability(analysis.getInputBaseDirectory(), "input")
-        errors += checkDirForReadabilityAndExecutability(analysis.getOutputBaseDirectory(), "output")
+        // The checks are done before the readability tests because we need to check for the raw configuration values
+        // as the next check will already use translated value
 
-        // Out dir needs to be writable
-        if (!FileSystemAccessProvider.instance.isWritable(analysis.getOutputBaseDirectory()))
-            errors << (String) "The output was not writeable at path ${analysis.getOutputBaseDirectory()}."
+        String valueInDir = analysis.configuration.configurationValues.get(ConfigurationConstants.CFG_INPUT_BASE_DIRECTORY, "").value
+        String valueOutDir = analysis.configuration.configurationValues.get(ConfigurationConstants.CFG_OUTPUT_BASE_DIRECTORY, "").value
+        
+        if (!valueInDir && !valueOutDir) {
+            errors << "Both the input and output base directories are not set. You must set at least inputBaseDirectory or outputBaseDirectory."
+
+        } else {
```

Comment:
To have less nesting, why not `} else if (...) {`?

Reply:
Sorry, don't get it. Why introduce another if?